bin/nvcc_wrapper

```diff
@@ -202,11 +202,11 @@ do
     cuda_args="$cuda_args $1"
     ;;
   #Handle unsupported standard flags
-  --std=c++1y|-std=c++1y|--std=c++1z|-std=c++1z|--std=gnu++1y|-std=gnu++1y|--std=gnu++1z|-std=gnu++1z|--std=c++2a|-std=c++2a|--std=c++17|-std=c++17)
+  --std=c++1y|-std=c++1y|--std=gnu++1y|-std=gnu++1y|--std=c++1z|-std=c++1z|--std=gnu++1z|-std=gnu++1z|--std=c++2a|-std=c++2a)
```

Comment:
This isn't going to work for compilers that support C++17 with CUDA < 11. CMake is going to do a version test expecting to find C++17 support and will then fail the feature tests.

I think we are going to have to move c++17 into its own special case and make this version-dependent.